cypress/support/pages/backoffice/product-measurement-unit/create/product-measurement-unit-create-page.ts

```diff
@@ -0,0 +1,30 @@
+import { autoWired } from '@utils';
+import { inject, injectable } from 'inversify';
+
+import { BackofficePage } from '@pages/backoffice';
+import { ProductMeasurementUnitCreateRepository } from './product-measurement-unit-create-repository';
+
+@injectable()
+@autoWired
+export class ProductMeasurementUnitCreatePage extends BackofficePage {
+  @inject(ProductMeasurementUnitCreateRepository) private repository: ProductMeasurementUnitCreateRepository;
+
+  protected PAGE_URL = '/product-measurement-unit-gui/index/create';
+
+  interceptCreateFormSubmit = (alias: string): void => {
+    cy.intercept('POST', this.PAGE_URL + '*').as(alias);
+  }
```

Comment:
not sure that it's needed. In your test, you submit the form by clicking 'Save' (or similar) button. This click action will trigger synchronous new page navigation, and Cypress automatically waits for the page’s load event. 

Some backoffice tables are tricky and execute a couple of async requests, but in this test I think we can try without it.